README.md

```diff
@@ -20,44 +20,33 @@ work with this repository is to do the following:
 
 ### Cloning the Repo
 
-```bash 
-# rather than ~/go for everything, provide separate gopaths
-mkdir $HOME/gopaths
-``` 
-
-It is fine to have `~/gopaths` next to `~/go` if you have some legacy projects.
+Rather than ~/go for everything, provide separate gopaths. For this example we will
+use $HOME/gopaths as the root of our projects and consoleoperator as our project directory.
 
-Now, create a `dir` under `~gopaths` to hold the project:
 
 ```bash 
-mkdir $HOME/gopaths/consoleoperator 
-```
+# rather than ~/go for everything, provide separate gopaths for this example we will
```

Comment:
Let's drop this comment since it's a dup from above

Reply:
finally got to it.